doc/tutorials/agl-vms.rst

```diff
@@ -145,19 +145,24 @@ Service OS
 
       # swupd autoupdate --disable
 
-#. This demo setup uses a specific release version (26200) of Clear
+#. This demo setup uses a specific release version (31080) of Clear
    Linux OS which has been verified to work with ACRN. In case you
    unintentionally update or change the Clear Linux OS version, you can
    fix it again using::
 
-      # swupd verify --fix --picky -m 26200
+      # swupd verify --fix --picky -m 31080
 
-#. Use the ``swupd bundle-add`` command and add needed Clear Linux
-   OS bundles::
+#. Use `acrn_quick_setup.sh <https://github.com/projectacrn/acrn-hypervisor/blob/84c2b8819f479c5e6f4641490ff4bf6004f112d1/doc/getting-started/acrn_quick_setup.sh>`_ 
+   to automatic installation ACRN::
+      
+      # sh acrn_quick_setup.sh -s 31080 -i 
+
+#. After installation, the system will start up again automatically
+
+#. Reboot the system, choose "ACRN Hypervisor" and launch Clear Linux OS
+   SOS. If the EFI boot order is not right, use :kbd:`F10`
```

Comment:
Please remove this instance and all possible instances of SOS and UOS and replace with Service VM / User VM.